lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -66,6 +66,17 @@
   .form-actions {
     padding-top: gutter();
   }
+
+  &.-signup {
+    width: 30%;
+    min-width: 256px;
+    margin: 1.5%;
+    display: inline-block;
+
+    .beta-card__row:first-of-type {
+      margin-top: 0;
+    }
+  }
```

Comment:
This has nothing to do with the gallery / triad format. When the campaign signup card has no header, it leaves a margin at the top which creates an empty row of white rather than being flush with the border. Heres an example:

<img width="321" alt="screen shot 2016-09-20 at 10 24 26 am" src="https://cloud.githubusercontent.com/assets/897368/18674225/6ff976ba-7f1c-11e6-9071-00c34b333987.png">

I renamed this class from `-signup` to `-headerless` to better reflect what the style was doing